roles/django_manage_py_cron/templates/manage_py_command.sh

```diff
@@ -0,0 +1,8 @@
+# This is a script that makes runnning manage.py inside a virtualenv from
+# cron easier.
+# It does a few thigns:
+#     1. It wraps comands in `output-on-error`
+#     2. It calls the manage.py with the virtualenv's python
+# It required a var `{{ project_name }}` and for the project to be at
+# /var/www/{{ project_name }}
+/usr/local/bin/output-on-error /var/www/{{ project_name }}/env/bin/python /var/www/{{ project_name }}/code/manage.py "$@"
```

Comment:
I might look at moving this into https://github.com/DemocracyClub/ansible-role-commonscripts at some point as it is probably a useful script for all/most of our projects. Lets ignore that for now though.

Reply:
Yep, I had that in mind when I wrote it, but figured it's better to leave it here until we've tested it a little more.